src/drift/transformations/sequential.py

```diff
@@ -0,0 +1,29 @@
+from .base import Transformation
+
+import pandas as pd
+from typing import TYPE_CHECKING
+
+
+if TYPE_CHECKING: 
+    from neuralforecast.common._base_windows import BaseWindows
+
+
+class SequentialTransformation(Transformation):
```

Comment:
I think we can name this NeuralForecastWrapper, or something similar?
I think all of the different libraries will require slightly different data preparation steps, so we can't generalize this (Maybe to `StatsForecast` models?)

Reply:
Yes, done!